army schema work/1 table.sql

```diff
@@ -0,0 +1,42 @@
+use ArmyDB
+go
+
+drop table if exists dbo.Soldier
+go 
+
+create table dbo.Soldier
+(
+FirstName VARCHAR(20) not null
+    CONSTRAINT ck_Soldier_FirstName_cannot_be_blank CHECK (FirstName <> ''),
+LastName VARCHAR(20) not null
+    CONSTRAINT ck_Soldier_LastName_cannot_be_blank CHECK (LastName <> ''),
+SSN VARCHAR(10) not null 
+    CONSTRAINT u_Soldier_SSN_must_be_unique unique,
+    CONSTRAINT ck_Soldier_SSN_have_9_digits CHECK (SSN like '%%%%%%%%-%'),
+DateBorn date not null
+    CONSTRAINT ck_Soldier_DateBorn_cannot_be_blank CHECK (DateBorn <> ''),
+City VARCHAR(20) not null
+    CONSTRAINT ck_Soldier_City_cannot_be_blank CHECK (City <> ''),
+EnlistmentDate date not null 
+    CONSTRAINT ck_Soldier_EnlistmentDate_must_be_before_current_date CHECK (EnlistmentDate < GETDATE()),
+    CONSTRAINT ck_Soldier_EnlistmentDate_cannot_be_blank CHECK (EnlistmentDate <> ''),
+MilitaryBranch VARCHAR(12) not null
+    CONSTRAINT ck_Soldier_MilitaryBranch_cannot_be_blank CHECK (MilitaryBranch <> ''),
+    CONSTRAINT ck_Soldier_MilitaryBranch_is_navy_land_or_air CHECK (MilitaryBranch in ('Navy', 'Air Force', 'Ground Force')),
+Rank VARCHAR(25) not null 
+    CONSTRAINT ck_Soldier_Rank_cannot_be_blank CHECK (Rank <> ''),
+IQLevel int not null CONSTRAINT ck_Soldier_IQLevel_between_0_and_300 CHECK (IQLevel between 0 and 300),
```

Comment:
-- LL do you know for sure that 300 is the highest level? I don't think there officially is a limit to how high an iq can be so even though the highest level recorded was only in the 200's, you don't want to limit it to 300 in case it would ever go higher

Reply:
-- LL you are correct but since there is no official limit, there's no reason to limit it, instead just say it has to be greater than 0